compiler/z/codegen/ControlFlowEvaluator.cpp

```diff
@@ -3101,7 +3101,7 @@ OMR::Z::TreeEvaluator::ternaryEvaluator(TR::Node *node, TR::CodeGenerator *cg)
       if (TR::Compiler->target.cpu.getSupportsArch(TR::CPU::z15))
          {
          auto mnemonic = trueVal->getOpCode().is8Byte() ? TR::InstOpCode::SELGR : TR::InstOpCode::SELR;
-         generateRRFInstruction(cg, mnemonic, node, trueReg, trueReg, falseReg, getMaskForBranchCondition(TR::InstOpCode::COND_BER));
+         generateRRFInstruction(cg, mnemonic, node, trueReg, falseReg, trueReg getMaskForBranchCondition(TR::InstOpCode::COND_BER));
```

Comment:
We're missing a comma after `trueReg` here.

Reply:
Fixed: bfb09be0e890b08a6dca12604ffd9314136b0121